.github/workflows/production.yml

```diff
@@ -40,6 +43,8 @@ jobs:
           POSTHOG_TIMEOUT_MS: 1000
           POSTHOG_PROJECT_ID: ${{ secrets.POSTHOG_PROJECT_ID_PROD }}
           POSTHOG_PROJECT_API_KEY: ${{ secrets.POSTHOG_PROJECT_API_KEY_PROD }}
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN_PROD }}
+          SENTRY_ENV: ${{ secrets.MITOPEN_ENV_PROD }}
```

Comment:
Actions environments would be an improvement here so we are not having env vars that include the environment name - the job would run in the environment context. Outside of the scope of this PR though, given there are others.

Reply:
For us, it is quite canonical. All of our repos have it, and Doof updates it automatically during release via https://github.com/mitodl/release-script/blob/b44bb831e78a5450e3b09e502fd8733b454157f0/version.py#L120